addons/l10n_cl_website_sale/views/templates.xml

```diff
@@ -0,0 +1,92 @@
+            <label t-attf-class="col-form-label #{mode[1] == 'shipping' and 'label-optional' or ''}"
+                   for="l10n_cl_dte_email">DTE Email
+            </label>
+            <input type="email" name="l10n_cl_dte_email"
+                   t-attf-class="form-control #{error.get('l10n_cl_dte_email') and 'is-invalid' or ''}"
+                   t-att-value="'l10n_cl_dte_email' in checkout and checkout['l10n_cl_dte_email']"/>
+        </div>
+    </template>
+
+    <template id="l10n_cl_info_2" name="Chilean partner cl info 2">
+        <t t-set='vat_warning' t-value="'vat' in checkout and checkout['vat'] and not can_edit_vat"/>
+        <t t-if="mode == ('new', 'billing') or (mode == ('edit', 'billing') and (can_edit_vat or 'vat' in checkout and checkout['vat']))">
+            <div t-attf-class="mb-3 #{error.get('company_name') and 'o_has_error' or ''} col-lg-6 mb-0">
+                <label class="col-form-label fw-normal label-optional" for="company_name">Company Name</label>
+                <input type="text" name="company_name"
+                       t-attf-class="form-control #{error.get('company_name') and 'is-invalid' or ''}"
+                       t-att-value="'commercial_company_name' in checkout and checkout['commercial_company_name'] or 'company_name' in checkout and checkout['company_name']"
+                       t-att-readonly="'1' if vat_warning else None"/>
+                <small t-if="vat_warning" class="form-text text-muted d-block d-lg-none">Changing company name is not
+                    allowed once document(s) have been issued for your account. Please contact us directly for this
+                    operation.
+                </small>
+            </div>
+            <div t-attf-class="mb-3 #{error.get('vat') and 'o_has_error' or ''} col-lg-6 div_vat mb-0">
+                <label class="col-form-label fw-normal label-optional" for="vat">TIN / VAT</label>
+                <input type="text" name="vat" t-attf-class="form-control #{error.get('vat') and 'is-invalid' or ''}"
+                       t-att-value="'vat' in checkout and checkout['vat']"
+                       t-att-readonly="'1' if vat_warning else None"/>
+                <small t-if="vat_warning" class="form-text text-muted d-block d-lg-none">Changing VAT number is not
+                    allowed once document(s) have been issued for your account. Please contact us directly for this
+                    operation.
+                </small>
+            </div>
+            <div t-if="vat_warning" class="col-12 d-none d-lg-block mb-1">
+                <small class="form-text text-muted">Changing company name or VAT number is not allowed once document(s)
+                    have been issued for your account. Please contact us directly for this operation.
+                </small>
+            </div>
+        </t>
+    </template>
+
+    <template id="address_l10n_cl" inherit_id="website_sale.address">
```

Comment:
@hea-bmya Is there a way to keep the address view unchanged, but do everything in the B2B ones?  So in the case of B2C you would always do a boleta to consumidor final?  @Danisan